### PR TITLE
[ZPI-321] Correcting labels in attraction categories

### PR DESCRIPTION
**Pull request by AdrianBroniecki**

### CODE REVIEW

File: app/views/AddingTripPointView.tsx [hunk 1]
```diff
@@ -177,7 +178,8 @@ const AddingTripPointView = () => {
         );
         setTripPointCategory(
           getCategoryByName(
-            destinationDetails.superCategory?.name ?? DEFAULT_CATEGORY_NAME,
+            destinationDetails.superCategory?.name ??
```
Comment (bartoszgableta) [inline app/views/AddingTripPointView.tsx:181]: Is this needed? From what i understand the function already checks for the superCategory name

File: app/views/TripPointDetailsView.tsx
Comment (bartoszgableta) [inline app/views/TripPointDetailsView.tsx:1]: Here once again the changes were overriden. I can fix that if you want me to

Comment (AdrianBroniecki) [reply app/views/TripPointDetailsView.tsx:1]: Solved

File: app/views/AddingTripPointView.tsx (hunk 1 above)
Comment (AdrianBroniecki) [reply app/views/AddingTripPointView.tsx:181]: It returns category with id - in request we need the whole category object. Also sometimes it happens that superCategory is null and we have to search for category in category list.

**Review by bartoszgableta — APPROVED** [review]

**Pull request merged**